### PR TITLE
Update change-log-sparkpost-enterprise.md

### PR DESCRIPTION
**Pull request by Irinadoliov**
Updated for self-service bounce domains in Enterprise in the release the week of July 24

### CODE REVIEW

File: articles/tech-resources/change-log-sparkpost-enterprise.md [hunk 1]
```diff
@@ -7,6 +7,15 @@ notification: "This Change Log is for SparkPost Enterprise Only"
 A running log of new features and capabilities that have been rolled out on the SparkPost Enterprise service. The API reference can be found [here](https://developers.sparkpost.com/api).
 
 
+### (Week of) 28 July, 2017 
+
+* **Enhancement:** Self Service for Bounce (Return-Path) Domains in SparkPost Enterprise
+  * Enterprise customers can now create and verify Bounce Domains (aka Return-Path Domains) in addition to Sending (aka FROM) Domains via an API without calling their TAM or submitting a Support ticket.
```
Comment (aymg) [inline articles/tech-resources/change-log-sparkpost-enterprise.md]: `Support ticket` -> `support ticket`

File: articles/tech-resources/change-log-sparkpost-enterprise.md [hunk 2]
```diff
@@ -7,6 +7,15 @@ notification: "This Change Log is for SparkPost Enterprise Only"
 A running log of new features and capabilities that have been rolled out on the SparkPost Enterprise service. The API reference can be found [here](https://developers.sparkpost.com/api).
 
 
+### (Week of) 28 July, 2017 
+
+* **Enhancement:** Self Service for Bounce (Return-Path) Domains in SparkPost Enterprise
+  * Enterprise customers can now create and verify Bounce Domains (aka Return-Path Domains) in addition to Sending (aka FROM) Domains via an API without calling their TAM or submitting a Support ticket. 
+  * Enterprise customers can call the Sending Domains API to Create the Domain, CNAME it to customerID.mail.e.sparkpost.com, and call the API again to verify that the DNS entry is correct. This will allow OOB bounces to be processed correctly.
+  * Service providers that have auto_verify turned on will be flagged as "CNAME verified" without doing the DNS check as we assume the customer is doing the correct DNS updates.
+  * This [Support article](https://www.sparkpost.com/docs/tech-resources/custom-bounce-domain/) describes the process for creating Bounce Domains, including the create and verify endpoints.
```
Comment (aymg) [inline articles/tech-resources/change-log-sparkpost-enterprise.md]: `Support article` -> `support article`

File: articles/tech-resources/change-log-sparkpost-enterprise.md [hunk 3]
```diff
@@ -7,6 +7,15 @@ notification: "This Change Log is for SparkPost Enterprise Only"
 A running log of new features and capabilities that have been rolled out on the SparkPost Enterprise service. The API reference can be found [here](https://developers.sparkpost.com/api).
 
 
+### (Week of) 28 July, 2017
```
Comment (aymg) [inline articles/tech-resources/change-log-sparkpost-enterprise.md]: `(Week of)`- is this an accepted pattern for the changelog format?

File: articles/tech-resources/change-log-sparkpost-enterprise.md [hunk 4]
```diff
@@ -7,6 +7,15 @@ notification: "This Change Log is for SparkPost Enterprise Only"
 A running log of new features and capabilities that have been rolled out on the SparkPost Enterprise service. The API reference can be found [here](https://developers.sparkpost.com/api).
 
 
+### (Week of) 28 July, 2017 
+
+* **Enhancement:** Self Service for Bounce (Return-Path) Domains in SparkPost Enterprise
+  * Enterprise customers can now create and verify Bounce Domains (aka Return-Path Domains) in addition to Sending (aka FROM) Domains via an API without calling their TAM or submitting a Support ticket. 
+  * Enterprise customers can call the Sending Domains API to Create the Domain, CNAME it to customerID.mail.e.sparkpost.com, and call the API again to verify that the DNS entry is correct. This will allow OOB bounces to be processed correctly.
```
Comment (aymg) [inline articles/tech-resources/change-log-sparkpost-enterprise.md]: `Create` -> `create`

File: articles/tech-resources/change-log-sparkpost-enterprise.md (hunk 3 above)
Comment (Irinadoliov) [inline articles/tech-resources/change-log-sparkpost-enterprise.md]: "Accepted"? I dunno, I created this page and have maintained it from the start so I accept it. :-). Upgrades to Enterprise Tenants will happen that week so they happen on different days for different customers. I can change it to be that Friday if "week of" seems odd.

Comment (aymg) [inline articles/tech-resources/change-log-sparkpost-enterprise.md]: It works for me! I just didn't see it anywhere else :) lets keep it 👍

**Pull request merged**